api-docs/index.html

```diff
@@ -6867,6 +6870,72 @@ <h3 id='params-7'>Params</h3>
+
+<span class="kr">const</span> <span class="nx">clientOptions</span> <span class="o">=</span> <span class="p">{</span>
+  <span class="na">network</span><span class="p">:</span> <span class="nx">network</span><span class="p">.</span><span class="nx">type</span><span class="p">,</span>
+  <span class="na">port</span><span class="p">:</span> <span class="nx">network</span><span class="p">.</span><span class="nx">rpcPort</span><span class="p">,</span>
+  <span class="na">apiKey</span><span class="p">:</span> <span class="s1">'api-key'</span>
+<span class="p">}</span>
+
+<span class="kr">const</span> <span class="nx">client</span> <span class="o">=</span> <span class="k">new</span> <span class="nx">NodeClient</span><span class="p">(</span><span class="nx">clientOptions</span><span class="p">);</span>
+
+<span class="p">(</span><span class="nx">async</span> <span class="p">()</span> <span class="o">=&gt;</span> <span class="p">{</span>
+  <span class="kr">const</span> <span class="nx">result</span> <span class="o">=</span> <span class="nx">await</span> <span class="nx">client</span><span class="p">.</span><span class="nx">execute</span><span class="p">(</span><span class="s1">'getdnssecproof'</span><span class="p">,</span> <span class="p">[</span> <span class="nx">name</span> <span class="p">]);</span>
+  <span class="nx">console</span><span class="p">.</span><span class="nx">log</span><span class="p">(</span><span class="nx">result</span><span class="p">);</span>
+<span class="p">})();</span>
+</code></pre><pre class="highlight json tab-json"><code><span class="p">{</span><span class="w">
+  </span><span class="s2">"zones"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="err">...</span><span class="p">]</span><span class="w">
+</span><span class="p">}</span><span class="w">
+</span></code></pre>
+<p>Query for and build a Handshake DNSSEC ownership proof.
+This is the same proof that is included in a name claim. See
+the <a href="#sendclaim">sendclaim</a> RPC method to build a transaction that includes
+the DNSSEC ownership proof. The <code>estimate</code> param is used to
+validate the proof. If set to <code>false</code>, it will strictly require
+the proof to be valid. If set to <code>true</code>, the proof will be build
+and returned to the caller, even if it is invalid. The default
+value for <code>estimate</code> is <code>false</code>.</p>
+
+<p>A Handshake DNSSEC ownership proof is a more strict subset of
+a DNSSEC proof. Each parent/child zone must operate through a series
+of <code>DS-&gt;DNSKEY</code> relationships, no <code>CNAME</code>s or wildcards are allowed,
+each label separation (<code>.</code>) must behave like a zone cut (with an
+appropriate child zone referral) and weak cryptography is not allowed.
+This means that <code>SHA1</code> digests for <code>DS</code> hashes are not allowed,
```

Comment:
I think you can take SHA1 out of code style. Sometimes the renderer screws up line breaks:

![Screen Shot 2020-05-05 at 9 13 49 AM](https://user-images.githubusercontent.com/2084648/81070138-d9578000-8eb0-11ea-836f-f5d796cbd61e.png)